nix/test-environments/test-script.py

```diff
@@ -0,0 +1,300 @@
+import re
+import shlex
+import json
+import time
+from dataclasses import dataclass
+from typing import List
+from test_driver.errors import RequestedAssertionFailed # type: ignore
+
+# --- Configuration ---
+@dataclass
+class TestConfig:
+    """Centralized configuration for the integration test."""
```

Comment:
Quite a lot of constants here, would it not be better to provide them from the `nix` side? Not sure what's the preferred way when writing `nixos` integration tests, might be perfectly fine this way

Reply:
I wonder where we can take them from the generated by nix config files (sequencer and reporter config json)